src/Components/Nav/NavDropDown.js

```diff
@@ -6,7 +6,7 @@ import { Link } from 'react-router-dom';
 const NavDropDown = ({ options }) => {
     return (
         <UncontrolledDropdown nav inNavbar>
-            <DropdownToggle nav caret>
+            <DropdownToggle nav caret className="nav-option">
```

Comment:
We use this class here but is defined in another css component, should we group all nav css into one file?